.github/workflows/podman-desktop-e2e-remote-windows.yaml

```diff
@@ -105,8 +134,10 @@ jobs:
             --conn-details-output '/workspace' \
             --windows-version '${{ matrix.windows-version }}' \
             --windows-featurepack '${{ matrix.windows-featurepack }}' \
-            --vmsize '${{ env.HOST_FLAVOR }}' \
```

Comment:
Also I don't see any other way to set the flavor, what's up with that?

Reply:
No need. I am searching for any sufficient machine with given cpus/memory and nested virt.